docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -17,19 +17,21 @@ In KQL, this translates to any event with the `agent.type` set to either:
 
 To access events that can be visually analyzed:
 
-1. Select *Explore* -> *Hosts* -> *Events*. A list of all your hosts' events appears at the bottom of the Hosts page.
+1. Select *Explore* -> *Hosts*, then select the *Events* tab. A list of all your hosts' events appears at the bottom of the Hosts page.
 
-2. Create a KQL query that filters all `endpoint` detected events by entering either `agent.type:"endpoint" and process.entity_id : *` or  `agent.type:"winlogbeat" and event.module: "sysmon" and process.entity_id : *` into the KQL search bar, and then selecting **Update**.
+2. Create a KQL query that filters all `endpoint` detected events by entering either query into the KQL search bar, and then selecting **Update**:
```

Comment:
suggestion. Replace: 
- "Create a KQL query that filters all `endpoint` detected events by entering either query into the KQL search bar, and then selecting **Update**:
With:
- "Search for visualizable events by entering either of the following queries into the KQL search bar, and selecting **Update**:"

Reply:
A slight revision on Ben's suggestion: 

"Filter events that can be visually analyzed by entering either of the following queries in the KQL search bar, then selecting **Update:**"